### PR TITLE
Use private IPs when interacting with instances

### PR DESCRIPTION
**Pull request by tlunter**
If a subnet requests a public IP address, but we still want to interact over private IPs, enable this.

### CODE REVIEW

**Comment by tlunter** [comment]
@bfulton @adamjt @nahiluhmot

File: lib/aerosol/instance.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def private_ip_address
   end
 
   def address
-    if public_hostname.blank?
+    if launch_configuration.try(:always_use_private_address) || public_hostname.blank?
```
Comment (nahiluhmot) [inline lib/aerosol/instance.rb]: Why do we need to `.try` this?

Comment (tlunter) [inline lib/aerosol/instance.rb]: I assumed I was breaking some invariant, but maybe it was just protecting bad stubs. Likely that.

**Comment by tlunter** [comment]
@nahiluhmot no more `.try`.

**Comment by adamjt** [comment]
+1

**Comment by bfulton** [comment]
xrj5we jpg

**Comment by nahiluhmot** [comment]
:+1:

**Pull request merged**